README.md

```diff
@@ -492,11 +492,17 @@ When `false`, the following will cause an error:
 <div styleName='foo bar' />
 ```
 
-#### `errorWhenNotFound`
+#### `handleNotFoundStyleName`
 
-Default: `true`.
+Default: `'throw'`.
 
-Throws an error when `styleName` cannot be mapped to an existing CSS Module.
+Define the desired action when `styleName` cannot be mapped to an existing CSS Module.
+
+Available options
+
+- `'throw'` will throw an error
+- `'log'` will log a warning to the console
+- `'ignore'` will silently ignore it
```

Comment:
Actually I think the wording used to define each option is a bit redundant, but if you think it is understandable it's good enough :)